app/routes/entries/browse/$initialLetters/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { type LoaderArgs, redirect } from "@remix-run/server-runtime";
+
+export async function loader({ request, params }: LoaderArgs) {
+    return redirect(`${params.initialLetters}/1`);
```

Comment:
For me at least, this does not appear to work once I've removed the `/1` over in `/app/routes/entries/index.tsx` (see comment there). I think we need the full path here:

```ts
return redirect(`/entries/browse/${params.initialLetters}/1`)
```

Reply:
Done